### PR TITLE
add snapstore installtion command

### PR DESCRIPTION
**Pull request by purveshpatel511**
Hey, @sharkdp, I released this package as a snap on snapstore. adding snap installtion method in readme file also updated changelog file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
 - Completely new CI/CD system via GitHub Actions, see #120 (@rivy)
 
+## Packaging
+
+- `pastel` is now available on snapstore, see #130 (@purveshpatel511)
+
```
Comment (sharkdp) [inline CHANGELOG.md:21]: please add this to "unreleased", not to an existing version

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 [![](https://img.shields.io/github/v/release/sharkdp/pastel?colorB=d7a400&style=flat-square)](https://github.com/sharkdp/pastel/releases)
 [![](https://img.shields.io/crates/l/pastel.svg?colorB=ff7155&style=flat-square)](https://crates.io/crates/pastel)
 [![](https://img.shields.io/crates/v/pastel.svg?colorB=ff69b4&style=flat-square)](https://crates.io/crates/pastel)
+[![pastel](https://snapcraft.io/pastel/badge.svg)](https://snapcraft.io/pastel)
+[![pastel](https://snapcraft.io/pastel/trending.svg?name=0)](https://snapcraft.io/pastel)
```
Comment (sharkdp) [inline README.md:8]: I'd rather not add packaging-specific icons here (or anywhere).

File: README.md [hunk 3]
```diff
@@ -167,6 +169,15 @@ git clone https://github.com/sharkdp/pastel
...
 
+### Via snap package
+
+```
+sudo snap install pastel
+```
+
+[Get it from the Snap Store](https://snapcraft.io/pastel)
+
+
```
Comment (sharkdp) [inline README.md:180]: can we please move it below the "On MacOS" section?

**Comment by purveshpatel511** [comment]
@sharkdp suggested changes is done.

**Pull request merged**